src/components/backend-ai-registry-list.ts

```diff
@@ -227,22 +234,22 @@ class BackendAIRegistryList extends BackendAIPage {
    * */
   _addRegistry() {
     // somehow type casting is needed to prevent errors, unlike similar use cases in other files
-    const hostname = (<HTMLInputElement> this.shadowRoot.querySelector('#add-registry-hostname')).value;
-    const url = (<HTMLInputElement> this.shadowRoot.querySelector('#add-registry-url')).value;
-    const username = (<HTMLInputElement> this.shadowRoot.querySelector('#add-registry-username')).value;
-    const password = (<HTMLInputElement> this.shadowRoot.querySelector('#add-registry-password')).value;
-    const registerType = this.shadowRoot.querySelector('#select-registry-type').value;
-    const projectName = this.shadowRoot.querySelector('#add-project-name').value.replace(/\s/g, '');
-
-    if (!this.shadowRoot.querySelector('#add-registry-hostname').valid) {
+    const hostname = this.hostname.value;
+    const url = this.url.value;
+    const username = this.username.value;
+    const password = this.password.value;
+    const registerType = this.registerType.value;
+    const projectName = this.projectName.value.replace(/\s/g, '');
+
+    if (!this.shadowRoot.querySelector('#configure-registry-hostname').valid) {
       const validationMessage = this.shadowRoot.querySelector('#registry-hostname-validation');
```

Comment:
I think we can use the member variable instead of declaring and assigning local variables here.
And in L244, we already declared member variable for [querying `#configure-registry-hostname` in L54](https://github.com/lablup/backend.ai-webui/pull/1385/files#diff-4d9956da4424c56f3eca5366ff7485b320c865897c146fb86cc88c9f51d1239eR54), so let's use them instead of calling `querySelector` explicitly.

Reply:
I reflected it